### ISSUE TITLE
Killer feature suggestion

### ISSUE THREAD

**Issue by Petros-** [bomberstudios/artboard-manager#12]
It would be preetttty rad if my page titles updated the artboard names and then the order of the artboards (as you have it) determined the numbering. So say I'm running the artboard manager then all my artboards get the name of the page they're on and they get numbered according to their order visually on the artboard. To me that would make this plugin indispensible. No more selecting artboards to rename them — they just update automatically based on how you name the pages.

**Comment by vic-topcoder** [comment]
I'm confused - you want your page to be the template for all artboards on it? That rarely would be the case as you have a lot of different pages/states designed on an artboard. I think this was also mentioned in #5 and #7. If your request is to add the ability to use the current page as a regular expression in the naming of the artboards, then I'd say - that's good :) I use different names on different artboards and I wouldn't like to have the page name on ALL artboards though.

**Comment by Petros-** [comment]
Yeah, so if there's a little GUI for setup that let's you pick "Include page name in artboard name" then, yeah, that's what I'm talking about. It would be killer. There could also be a tickbox for "Include file name."

**Comment by vic-topcoder** [comment]
Yes, the request makes sense - a way to include the current page in the naming pattern for the artboards.

**Comment by Mikola35** [comment]
![image](https://user-images.githubusercontent.com/4017693/48952599-c6b6e880-ef53-11e8-9730-c3c7f416a5dc.png)

Or maybe @vic-topcoder you can use text layer before artboards row?

**Comment by Mikola35** [comment]
And if user change this text layer all artboards in this row should rename automatically
